mods/common/hotkeys/chat.yaml

```diff
@@ -1,21 +1,21 @@
 OpenTeamChat: Return
-	Description: Open Team Chat
+	Description: hotkey-description-OpenTeamChat
 	Types: Chat
-	Contexts: Player, Spectator
+	Contexts: hotkey-context-player, hotkey-context-spectator
```

Comment:
Using the translation key as internal ids feels inelegant IMO.

What if we make these internal ids (e.g. `Context: player, spectator`) and then map these into ftl as:
```
hotkey-contexts = 
    .player = Player
    .spectator = Spectator
    .chat-input = Chat Input
    .menu = Menu
    .editor = Editor
```

and then similarly for `hotkey-description`?

Reply:
hotkey descriptions should definitely not be mapped.

When it comes to context it definitely feels intelligent, but mapping would make it more magic. I'd agree to mapping only if the translation key would remain intact somewhere in yaml, though I don't know of an elegant way to do it

Reply:
If we wanted to use some opaque keys and then provide translations, we'd need to teach `HotkeysSettingsLogic` a mapping that would come from yaml such as:
```miniyaml
player: hotkey-context-player
spectator: hotkey-context-spectator
...
```

The downside of this is that we need some extra yaml for this mapping, and then we get into a faff with linting of the "context IDs". What if you use an ID that isn't defined in the mapping? What if the mapping lists a context that no hotkeys use?

Which is all doable but given it's only used by this one screen... there is something to be said for the current design where the contexts are just the names that show up for the user and there's no separate ID for them. Which sidesteps having to worry about those IDs at all.

Reply:
I wonder if contexts need explanations. I'm more leaning that they are self-explanatory enough

Reply:
For the description it can Imho stay, using a translation key there makes sense to me.

Reply:
Not sure how I feel about it, but changed to `Contexts: menu, player, spectator` and it uses the translation keys `hotkey-context.menu`, etc at runtime.